frontend/src/pages/MatchPage/index.jsx

```diff
@@ -44,13 +52,35 @@ export function MatchPage() {
   const [matches, setMatches] = useState([]);
   const [showModal, setShowModal] = useState(false);
 
-  const handleMatch = () => {
+  const handleMatch = async () => {
     // TODO: redux/saga api call to pull prospectsArray
     // TODO: redux/saga api call to pull user pref data for matchAlgo args
-
     // TODO: matchAlgo may include more args per user prefs for age/gender/proximity
-    const newMatches = matchAlgo(profile, prospectsArray);
-    // console.log("*** newMatches: ", newMatches)
+
+    var newMatches 
```

Comment:
Could probably change this to use 'let'?

Reply:
thanks Jeff, that tweak resolves it!